pkg/webhook/webhook.go

```diff
@@ -88,6 +70,15 @@ func (pwh *pravegaWebhookHandler) mutatePravegaManifest(ctx context.Context, p *
 }
 
 func (pwh *pravegaWebhookHandler) mutatePravegaVersion(ctx context.Context, p *pravegav1alpha1.PravegaCluster) error {
+	configMap := &corev1.ConfigMap{}
+	err := pwh.client.Get(ctx, types.NamespacedName{Name: p.Spec.SupportedUpgradePaths, Namespace: p.Namespace}, configMap)
+	if err != nil {
+		log.Println(err)
```

Comment:
Why are we logging as Println. We should log this with loglevel error.

Reply:
done